pkg/services/sqlstore/migrations/dashboard_snapshot_mig.go

```diff
@@ -66,6 +66,6 @@ func addDashboardSnapshotMigrations(mg *Migrator) {
 	}))
 
 	mg.AddMigration("Add encrypted dashboard json column", NewAddColumnMigration(snapshotV5, &Column{
-		Name: "dashboard_encrypted", Type: DB_Blob, Nullable: true,
+		Name: "dashboard_encrypted", Type: DB_Text, Nullable: true,
```

Comment:
Why was it had to be a text column exactly?

Reply:
It hasn't be fully decided yet - I wanted to experiment with it as a text field but I think considering the information I've found it might be best suitable as a blob - as it's not directly JSON but encrypted JSON.

According to official MySQL docs at least:
>Many encryption and compression functions return strings for which the result might contain arbitrary byte values. If you want to store these results, use a column with a VARBINARY or BLOB binary string data type. This avoids potential problems with trailing space removal or character set conversion that would change data values, such as may occur if you use a nonbinary string data type (CHAR, VARCHAR, TEXT).

https://dev.mysql.com/doc/refman/8.0/en/encryption-functions.html

Reply:
Thanks for clarifying. I don't see any upside to storing as TEXT even if it's JSON though? It's so easy to get encoding errors, which is why I made that issue.